client/packages/invoices/src/Prescriptions/LineEditView/PrescriptionLineEditForm.tsx

```diff
@@ -168,13 +183,31 @@ export const PrescriptionLineEditForm: React.FC<
         : 1;
 
     const numPacks = quantity / packSize;
-    debouncedAllocate(numPacks, Number(packSize));
+    debouncedAllocate(
+      numPacks,
+      Number(packSize),
+      quantityType === 'prescribed' ? inputUnitQuantity : prescribedQuantity
+    );
   };
 
-  const prescriptionLineWithNote = draftPrescriptionLines.find(l => !!l.note);
-  const note = prescriptionLineWithNote?.note ?? '';
+  const handlePrescribedQuantityChange = (inputPrescribedQuantity?: number) => {
+    if (!inputPrescribedQuantity) return;
```

Comment:
```suggestion
    if (inputPrescribedQuantity == null) return;
```

This one has caught me out a few times in the past -- if you just do`!inputPrescribedQuantity`, it'll return on *any* falsy value, including `0`. We need to accept "0", so check needs to be for explicitly non-values.

Just an annoying quirk of JS to always keep in mind ;)